dockerfiles/debian-buster-all/make_pg_buildext_parallel.patch

```diff
@@ -0,0 +1,17 @@
+--- /usr/bin/pg_buildext	2018-08-21 07:58:46.000000000 +0000
```

Comment:
If we can't apply it, we should remove the file.

Reply:
I'll fix this; this is _mostly_ for having builds actually use all cores.